Android/purchasesunity/src/main/java/com/revenuecat/purchasesunity/PurchasesWrapper.java

```diff
@@ -238,6 +239,41 @@ public static void checkTrialOrIntroductoryPriceEligibility(String jsonProducts)
 
     }
 
+    public static void invalidatePurchaserInfoCache() {
+        CommonKt.invalidatePurchaserInfoCache();
+    }
+
+    public static void setAttributes(String jsonAttributes) {
```

Comment:
I'm a bit confused by this - if we're already sending json and parsing keys, why are we sending keys and values separately? Why not just parse the JSON into dict directly?